### PR TITLE
Global elements basement

### PR DESCRIPTION
**Pull request by amberRrucker**
Looked over everything with David Stinemetze on www and everything looked to be in working order pending the small styling changes. In order for the basement to work and appear not wonky I brought in the rug changes. I can absolutely separate and rebase if we want to or we can close the rug PR and bundle these together.

### CODE REVIEW

**Comment by bighappyface** [comment]
Is this meant to replace/close #109 and #111?

**Comment by amberRrucker** [comment]
@bighappyface not #109 just #111. #109 doesnt belong to me :)

**Comment by amberRrucker** [comment]
The structural code is already within a PR submitted by other David in www.

File: styleguide/_themes/derek/scss/components/_rug.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.rug{
+  .rugSubhead{
+    font-size: 37px;
+    line-height: 24px;
+  }
+  &, *{
+    @extend .white;
+  }
+  .contactLink{
```
Comment (SyntaxStacks) [inline styleguide/_themes/derek/scss/components/_rug.scss]: `contact-link` no camel casing

File: styleguide/_themes/derek/scss/_modifiers.scss [hunk 2]
```diff
@@ -42,6 +42,14 @@ $body-bg:  #fff !default;
   right: 0;
 }
 
+.modal-noHeader.close{
```
Comment (SyntaxStacks) [inline styleguide/_themes/derek/scss/_modifiers.scss]: `modal-no-header`

File: styleguide/_themes/derek/scss/components/_rug.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.rug{
+  .rugSubhead{
```
Comment (SyntaxStacks) [inline styleguide/_themes/derek/scss/components/_rug.scss]: `rug-subhead`

File: styleguide/_themes/derek/scss/components/_basement.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.basement{
+  padding: 10px 0;
+  .basement-regionSelect{
```
Comment (bighappyface) [inline styleguide/_themes/derek/scss/components/_basement.scss]: No need to nest modular classes in the main module class. These classes should be out on their own as their names are unique.

File: styleguide/_themes/derek/scss/components/_rug.scss (hunk 1 above)
Comment (bighappyface) [inline styleguide/_themes/derek/scss/components/_rug.scss]: Hey @amberRrucker thanks for reading/implementing the MaintainableCSS style here. We are adding `sass-lint` in PR #105 and default configs force lowercase-hyphenated class naming form. We just talked about it and a group and are in agreement that we prefer lowercase-hyphenated to be more `sass-lint` compliant, and that is the only tweak we see necessary to make to MaintainableCSS for our needs.

Cool?

Comment (amberRrucker) [inline styleguide/_themes/derek/scss/components/_rug.scss]: @bighappyface sounds good to me!

File: styleguide/derek/_partials/_rug.ejs [hunk 5]
```diff
@@ -1,12 +1,48 @@
 <!--[rug]-->
-<div class="container rug">
+<div class="container-fluid bg-blue-gray half-padding">
+<div class="container rug hidden-xs hidden-sm">
```
Comment (bighappyface) [inline styleguide/derek/_partials/_rug.ejs]: Should the `hidden-xs hidden-sm` be moved to the `container-fluid` wrapper? I am still seeing a dark gray stripe at the bottom of the page in narrow viewport widths.

Comment (amberRrucker) [inline styleguide/derek/_partials/_rug.ejs]: absolutely. sorry! fixing now.

**Comment by bighappyface** [comment]
+1

**Pull request merged**